packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -163,6 +99,10 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
       console.warn(
         'sentry-vite-debug-id-upload-plugin not found in sentryPlugins! Cannot modify plugin - returning default Sentry Vite plugins',
       );
+
+    // resolving filesToDeleteAfterUpload here, because we return the original deletion plugin which awaits the promise
+    resolveFilesToDeleteAfterUpload(undefined);
```

Comment:
this is the ugly part about this PR: We always need to resolve the promise we pass to the vite plugin. So everywhere where we early return, in addition to the expected places.